indico/core/signals/event/registration.py

```diff
@@ -20,6 +20,10 @@
 
 _signals = Namespace()
 
+registration_modified = _signals.signal('registration_modified', """
```

Comment:
Right now `change` only includes personal data changes. What about making this signal specific to personal data changes too? Otherwise its a bit confusing if `change` is empty even though something did change